cellrank/tools/estimators/_base_estimator.py

```diff
@@ -280,8 +280,7 @@ def compute_absorption_probabilities(
         self,
         keys: Optional[Sequence[str]] = None,
         check_irred: bool = False,
-        norm_by_frequ: bool = False,
-        use_iterative_solver: Optional[bool] = None,
+        solver: Optional[str] = None,
         tol: float = 1e-5,
         use_initialization: bool = True,
```

Comment:
@Marius1311 since you've found out it doesn't help, can we remove this?

Reply:
yes.